source/compiler-core/slang-doc-extractor.cpp

```diff
@@ -717,8 +717,6 @@ SlangResult DocMarkupExtractor::extract(const SearchItemInput* inputs, Index inp
 {
     struct Entry
     {
-        typedef Entry ThisType;
```

Comment:
I dunno, just because it's not used doesn't mean it should be removed, despite warnings from gcc or elsewhere.

Reply:
I think that could be true elsewhere, but this particular case `Entry` is local to a function so doesn't take part in any interface, and within this function `Entry` is always going to be shorter (and clearer) than `Entry::ThisType`